libvmaf/src/feature/feature_extractor.c

```diff
@@ -30,9 +30,9 @@ extern VmafFeatureExtractor vmaf_fex_float_psnr;
 extern VmafFeatureExtractor vmaf_fex_float_ansnr;
 extern VmafFeatureExtractor vmaf_fex_float_adm;
 extern VmafFeatureExtractor vmaf_fex_float_vif;
-extern VmafFeatureExtractor vmaf_fex_integer_motion;
+extern VmafFeatureExtractor vmaf_fex_integer_motion2;
```

Comment:
if #594 is merged, I don't think it makes sense to rename these `*_motion2`, since they provide both. We could also use the same feature extractor for `motion3` in the future.